### DISCUSSION TITLE
[Esame Gennaio 2020] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#232] [Esami - Bertini @ Informatica]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/ecff29f8-3987-4938-afbd-7bf0b7294af9)

**Comment by LeoCosta02** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/ce92bce2-cf05-4492-81b2-87de34a57380)
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/82350d45-9284-4398-adda-415ffee0c775)
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/ea7b5947-abfe-46dc-b86c-8bf7c7852f21)

**Comment by LeoCosta02** [comment]
il punto 4 non l'ho svolto perchè non sono convinto del punto 3, l'indice della sommatoria non so se è giusta

**Comment by cornflexxx** [comment]
@LeoCosta02  Sono d'accordo con il primo punto come scritto anche  qui #116 .
$2)$ Calcolo $\mathbb{P}(Z_1=k | Z=x)$ : 
$$\mathbb{P}(Z_1=k | Z=x)=\frac {\mathbb{P}(Z_1=k \cap Z=x)}{\mathbb{P}(Z=x)}$$

Ma $Z=Z_1+Z_2$ quindi : 

$$\frac{\mathbb{P}(Z_1=k \cap Z_1+Z_2=x)}{\mathbb{P}(Z=x)}=\frac{\mathbb{P}(Z_1=k \cap Z_2=x-k)}{\mathbb{P}(Z=x)}=\frac{e^{-\lambda_1}\cdot\frac{\lambda_1 ^k}{k!}\cdot e^{-\lambda_2}\cdot\frac{\lambda_2^{x-k}}{x-k!}}{e^{-(\lambda_1+\lambda_2)}\cdot\frac{(\lambda_1+\lambda_2) ^x}{x!}}$$

Semplifico e raccolgo i $\lambda$:
$$\mathbb{P}(Z_1=k | Z=x)=\binom{x}{k}(\frac{\lambda_1}{\lambda_1+\lambda_2})^k \cdot (\frac{\lambda_2}{\lambda_1+\lambda_2})^{x-k}$$

Che possiamo ricondurre ad una binomiale.

$3)$  $$\mathbb{E}(Z_1|Z)=\sum_{k=0}^x k\cdot \mathbb{P}(Z_1=k | Z=x)= \sum_{k=0}^x k\cdot \binom{x}{k}(\frac{\lambda_1}{\lambda_1+\lambda_2})^k \cdot (\frac{\lambda_2}{\lambda_1+\lambda_2})^{x-k}= x \sum_{k=0}^x  \binom{x-1}{k-1}(\frac{\lambda_1}{\lambda_1+\lambda_2})^k \cdot (\frac{\lambda_2}{\lambda_1+\lambda_2})^{x-k}=\cdots$$

per comodità riscrivo $a=\frac{\lambda_1}{\lambda_1+\lambda_2}$ e $b=\frac{\lambda_2}{\lambda_1+\lambda_2}$
 
$$\cdots=x \sum_{k=0}^x  \binom{x-1}{k-1}a^k \cdot b^{x-k}=ax \sum_{k=0}^{x-1}  \binom{x-1}{k-1}a^{k-1} \cdot b^{x-k}=ax(a+b)^x=ax(a+1-a)^x=ax$$

In effetti è proprio il valore atteso di una $Bin(x,a)$.

$4)$ #249

**Comment by LeoCosta02** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/4213e071-a465-46b1-9abf-cf40ddf274a3)
@cornflexxx dove ho sottolineato con la penna rossa come fa a venirti sia al numero che al denominatore lo stesso elevamento a potenza, ovvero k-h ?

**Comment by cornflexxx** [comment]
@LeoCosta02 Il fattore prima del coefficiente binomiale le dovrebbe essere elevato alla h  e quindi ti sposti il denominatore elevato alla h a primo fattore e poi su quello a destra del coefficiente binomiale ti rimane il denominatore elevato alla k-h

**Comment by LeoCosta02** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/17d5f3ac-2b83-444a-b471-114f3f3a9182)
@cornflexxx Ho riscritto usando le tue variabili ,come devo fare? Scusami

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/user-attachments/assets/3d4f67d1-f978-4151-b490-7f85c7fe7626)
